server/game/challenge.js

```diff
@@ -199,7 +203,7 @@ class Challenge {
     }
 
     isUnopposed() {
-        return this.defenderStrength <= 0 && this.attackerStrength > 0;
+        return this.loserStrength <= 0 && this.winnerStrength > 0;
```

Comment:
They were already defined, we just weren't using them here. See https://github.com/cryogen/throneteki/blob/master/server/game/challenge.js#L144-L160